notebooks/official/training/tpuv5e_gemma_peft_finetuning_and_serving.ipynb

```diff
@@ -0,0 +1,1509 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #9.    SERVICE_ACCOUNT = "your-service-account"  # @param {type:"string"}
</pre><p>Set SERVICE_ACCOUNT = "[your-service-account]" instead. "[your-service-account]" will be replaced by a service account in the integration test.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/2775/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='27'/>

Reply:
All set